src/client-functions/selectors/selector-attribute-filter.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-undef */
```

Comment:
My bad, GH UI misleaded me. However, it's still makes sense to have a separate dir for filters.